CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v1.72.0
+------------------------------
+*November 1, 2019*
+
+### Updated
```

Comment:
There are two `Updated` sections.. but on that note, `Updated` isn't part of the changelog naming convention: https://keepachangelog.com/en/1.0.0/#how

Reply:
Should be `Changed` I think?